### PR TITLE
Changes to add Microsoft Edge compatibility

### PR DESCRIPTION
**Pull request by wench**
Changes needed so the version of Microsoft Edge (34.14291.1001.0) included in latest Windows 10 Insider Preview (14291) can sideload an unpacked xkit extension.

Small changes to xkit.js and Chrome\bridge.js so they work in Edge.

manifest.json could be shared between Edge and Chrome but the changes needed by Edge cause a warning when the extension is loaded in Chrome. I made a new edge specific manifest.js instead.

Added FIX_PERMISSIONS.cmd for Edge so the appropriate fs permissions can be set on the unpacked extension allowing Edge to load it.

Added edge to the build system.

### CODE REVIEW

File: Chrome/bridge.js [hunk 1]
```diff
@@ -12,6 +12,16 @@ var bridge_error_object;
 var xkit_storage = {};
 var bridge_ver = "2.1";
 
+// If chrome isn't defined, check for browser or msBrowser and use those instead. 
+if (typeof chrome === 'undefined') {
```
Comment (nightpool) [inline Chrome/bridge.js]: this code shouldn't be in Chrome/bridge.js. Instead, you should probably make an Edge specific bridge, right?

File: Edge/FIX_PERMISSIONS.cmd [hunk 2]
```diff
@@ -0,0 +1,3 @@
+REM Grant necessary fs permissions to the Edge sandbox user to allow access to the extension's directory
+cd /d %~dp0
+icacls "." /grant "*S-1-15-2-3624051433-2125758914-1423191267-1740899205-1073925389-3782572162-737981194":"(OI)(CI)(WDAC,WO,GE)"
```
Comment (nightpool) [inline Edge/FIX_PERMISSIONS.cmd]: This isn't in any of the Edge documentation... can someone confirm that's its necessary for XKit?

File: gulpfile.js [hunk 3]
```diff
@@ -167,12 +171,26 @@ gulp.task('copy:safari', ['clean:safari', 'lint'], function() {
 
 });
 
+gulp.task('copy:edge', ['clean:edge', 'lint'], function() {
+	var src = [].concat(
+		paths.scripts.core,
+		paths.css.core,
+		paths.vendor,
+		['Chrome/bridge.js', 'Chrome/icon.png', 'Edge/**/*']
```
Comment (nightpool) [inline gulpfile.js]: probably should copy icon to Edge or a shared directory (webkit?) (also see my comments about bridge above)

**Comment by nightpool** [comment]
we should probably decide whether we want to rename the chrome directory and share files or copy the bridge into the Edge directory. I'm leaning towards the latter.

**Comment by Wolvan** [comment]
Actually, we don't need to copy it. We could just have another partial bridge in edge and merge the two in the building process, basically what I did when I made the building of the firefox fork which I just so happen to forget the name of

File: Edge/FIX_PERMISSIONS.cmd (hunk 2 above)
Comment (wench) [inline Edge/FIX_PERMISSIONS.cmd]: All the edge extensions Microsoft released do it in their install scripts. Turn-Off-the-Lights has it too 
https://github.com/turnoffthelights/Turn-Off-the-Lights-Edge-extension/blob/master/Setup.cmd

**Comment by nightpool** [comment]
I would prefer not to do that for maintainability reasons. Either the code should be shared, or unique. Trying to do a mix of both seems bad.

**Comment by AprilSylph** [comment]
stale/superseded by #1645

**Pull request closed without merging**